app/models/build.rb

```diff
@@ -26,13 +26,14 @@ class Build < ActiveRecord::Base
       if: attribute
     )
   end
-  validates :docker_image_id, format: SHA256_REGEX, allow_nil: true
   validates :docker_repo_digest, format: DIGEST_REGEX, allow_nil: true
   validates :external_url, format: /\Ahttps?:\/\/\S+\z/, allow_nil: true
   validates :external_status, inclusion: Job::VALID_STATUSES, allow_nil: true
 
   before_create :assign_number
 
+  attr_accessor :docker_image_id
```

Comment:
Are you sure we still want to have this as an accessor, if it's not clear what's responsible for setting the value?

Reply:
it's not perfect, but a bit better than before ... next refactoring should get rid of it